distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -531,6 +532,39 @@ Text.replace self term:(Text | Regex) replacement:Text (case_sensitivity:Case_Se
             updated_regex = term.recompile case_sensitivity
             updated_regex.replace self replacement only_first
 
+## GROUP Text
+   ICON text
+   Removes the specified cleansings from the text.
```

Comment:
Maybe it should be phrased as "Applies the specified cleansings to the text"?

Reply:
Done